core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/type/PlatformType.java

```diff
@@ -51,37 +51,37 @@
  * @since 0.0.2
  */
 public enum PlatformType {
-  ILLUMINA("Illumina", "Flow Cell", "Lane", "Lanes", "nM", "ILLUMINA") {
+  ILLUMINA("Illumina", "Flow Cell", "Lane", "Lanes", "ILLUMINA") {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Flow Cell" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Lanes" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)